ppr-ui/src/components/common/DocumentId.vue

```diff
@@ -162,6 +162,10 @@ export default defineComponent({
       { immediate: true }
     )
 
+    watch(() => props.documentId, (val: string) => {
+      localState.documentIdModel = val
+    })
+
```

Comment:
Pretty minor but prop is already assigned to the local variable, do you know why a watcher is required here as well?

Reply:
Sounds good, thanks for looking into it.